modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/utils/odata.es.js

```diff
@@ -59,7 +87,7 @@ function valueParser(value, type) {
 		case PROPERTY_TYPES.COLLECTION:
 		case PROPERTY_TYPES.STRING:
 		default:
-			parsedValue = `'${value}'`;
+			parsedValue = escapeSingleQuotes(value);
```

Comment:
`escapeSingleQuotes` doesn't always return a string with quotes, could it be a problem? Before this change parsed value was always a string with quotes.